docs/releases/v5_0_0_3501-x86_64.md

```diff
@@ -0,0 +1,23 @@
+# libhkl-v5.0.0.3501-x86_64.tar.gz
+
+[v1.2.2](https://github.com/bluesky/libhkl/releases/tag/v1.2.2) | `libhkl-v5.0.0.3501-x86_64.tar.gz`
+
+package | version
+--- | ---
+sha256 | `TODO`
```

Comment:
Unresolved `TODO`

Reply:
Can't write the SHA until the library is built (will happen before the release).

Reply:
The tarball's SHA can be learned now:

```bash
user $ sha256sum tests/libhkl.tar.gz
8eb63eaf91d7551405876c6938ec94282f9945b763b327f393176692bc6df5da  tests/libhkl.tar.gz
```